### PR TITLE
chore(deps): update example to flutter 3.24.2

### PR DESCRIPTION
**Pull request by matiasleyba**
## Description

- [x] chore(deps): update example to flutter 3.24.2.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: .github/workflows/api.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
   build:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
     with:
-      dart_sdk: 3.4.3
+      dart_sdk: 3.5.0
```
Comment (marwfair) [inline .github/workflows/api.yaml:19]: Should this be 3.5.2 since that is the Dart version for Flutter 3.24.2?

Comment (matiasleyba) [reply .github/workflows/api.yaml]: yes, good catch

**Review by marwfair — DISMISSED** [review]
LGTM!

**Review by marwfair — APPROVED** [review]
LGTM!

**Pull request merged**